README.md

```diff
@@ -301,6 +301,14 @@ Default is `true`.
 This resource allows you to create and manage projects for GitHub repository.
 Default is `[]`.
 
+##### Webhooks Configuration
```

Comment:
could you also update the list of features at the top of the document (move webhooks from no implemented to extended?)

Reply:
Done 👌